Assets/Plugins/Source/Editor/Windows/CreatePackageWindow.cs

```diff
@@ -262,14 +262,14 @@ private async void StartCreatePackageAsync(UPMUtility.PackageType type)
             _createPackageCancellationTokenSource = new();
             _operationInProgress = true;
 
-            var progressHandler = new Progress<UnityPackageCreationUtility.CreatePackageProgressInfo>(value =>
+            var progressHandler = new Progress<FileUtility.CopyFileProgressInfo>(value =>
             {
                 var fileCountStrSize = value.TotalFilesToCopy.ToString().Length;
                 string filesCopiedStrFormat = "{0," + fileCountStrSize + "}";
                 var filesCopiedCountStr = String.Format(filesCopiedStrFormat, value.FilesCopied);
                 var filesToCopyCountStr = String.Format(filesCopiedStrFormat, value.TotalFilesToCopy);
 
-                _progress = value.SizeOfFilesCopied / (float)value.TotalSizeOfFilesToCopy;
+                _progress = value.BytesCopied / (float)value.TotalBytesToCopy;
```

Comment:
Is there ever a cursed world where TotalBytesToCopy is somehow 0? Like maybe a file with no contents and just a name?

Reply:
I don't believe so - but the change does not introduce a situation where that could happen - that possibility already exists. So worth pointing out, and I may make that change here and now - but technically it's outside the scope of the PR.